docs/source/serving/openai_compatible_server.md

```diff
@@ -157,8 +157,9 @@ vLLM will use guided decoding to ensure the response matches the tool parameter
 To enable this feature, you should set the following flags:
 * `--enable-auto-tool-choice` -- **mandatory** Auto tool choice. tells vLLM that you want to enable the model to generate its own tool calls when it 
 deems appropriate.
-* `--tool-call-parser` -- select the tool parser to use - currently either `hermes` or `mistral`. Additional tool parsers 
-will continue to be added in the future.
+* `--tool-call-parser` -- select the tool parser to use - currently either `hermes` or `mistral` or `internlm2_5` or `internlm2`. Additional tool parsers 
```

Comment:
Same comment as below - for internlm, since both `internlm2_5` and `internlm2` refer to the same format, we should just use `internlm` just like how `hermes` is used for multiple hermes models (and now Qwen 2.5 since it uses Hermes's format), and how `mistral` is used for various models as well.